src/controllers/SubredditsController.ts

```diff
@@ -0,0 +1,16 @@
+import { BodyParams, Controller, Post } from '@tsed/common';
+import { SubredditsCreation } from '../models/auth.types';
+
+import { SubredditsService } from '../services/SubredditsService';
+
+
+@Controller('/')
+export class SubredditsController {
+	constructor(private subredditsService: SubredditsService) {}
+
+	@Post('/subreddits/create')
+	public async createSubreddit(@BodyParams() 
+	subreddits: SubredditsCreation): Promise<SubredditsCreation> {
```

Comment:
This indendation is quite weird and it's hard to read. see if you can update it so it is more readable

Reply:
Indentation corrected